ocean_lib/data_provider/data_service_provider.py

```diff
@@ -382,14 +397,30 @@ def build_download_endpoint(provider_uri=None):
 
     @staticmethod
     def build_compute_endpoint(provider_uri=None):
-        return DataServiceProvider.build_endpoint('compute', provider_uri)
+        return DataServiceProvider.build_endpoint('computeStatus', provider_uri)
+
+    @staticmethod
+    def build_stop_compute(provider_uri=None):
+        return DataServiceProvider.build_endpoint('computeStop', provider_uri)
+
+    @staticmethod
+    def build_start_compute(provider_uri=None):
+        return DataServiceProvider.build_endpoint('computeStart', provider_uri)
+
+    @staticmethod
+    def build_delete_compute(provider_uri=None):
+        return DataServiceProvider.build_endpoint('computeDelete', provider_uri)
+
+    @staticmethod
+    def build_fileinfo(provider_uri=None):
+        return DataServiceProvider.build_endpoint('fileinfo', provider_uri)
 
     @staticmethod
     def get_initialize_endpoint(service_endpoint):
         parts = service_endpoint.split('/')
         if parts[-2] == 'services':
             base_url = '/'.join(parts[:-2])
-            return f'{base_url}/services/initialize'
+            return "GET", f'{base_url}/services/initialize'
```

Comment:
why is the `initialize` endpoint handled differently? Can't the logic from fileinfo, startCompute etc. be reused?